### DISCUSSION TITLE
기본 입력기를 끄고 싶습니다

### DISCUSSION THREAD

**Discussion by oxmhpark** [naverz/zepeto-studio-kor#880] [5. Q&A]
> 같은 내용을 이슈로도 작성했습니다. #873 

**제페토 월드 패키지 버전을 알려주세요. (Unity > Window > Package Manager 에서 버전 확인이 가능합니다.)
ZEPETO.World 1.4.1

**현재 설치되어 있는 제페토 앱 버전을 알려주세요. (제페토앱 > MY > 설정 > 버전 에서 버전 확인이 가능합니다.)
3.13.5

**어떤 기능을 구현하려고 했나요?
1. 기본 컨트롤러를 끄고 터치 제스처(스와이프 업/다운/레프트/라이트)로 캐릭터를 움직이고 싶습니다. (그리드 기반)

**어떤 문제가 발생 하였나요?
1. 내용이 없는 `InputActionAsset`을 새로 생성했습니다.
2. 기본 제공되는 `EventSystem` 프리팹을 복제해서 1번에서 생성한 `InputActionAsset`을 할당했습니다.
3. 씬에 존재하는 `ZepetoPlayers` 오브젝트를 다음과 같이 설정했습니다:
    - 1, 2번에서 만든 `InputActionAsset`과 `EventSystem`을 할당했습니다.
    - `UseDefaultVirtualController` 체크박스를 껐습니다.
4. 씬을 실행하면 다음과 같은 상태가 됩니다:
    - 씬에 생성된 `EventSystem(Clone)` 오브젝트는 2번에서 작업한대로 설정되어 있습니다.
    - `Virtual Controller`는 보이지 않으나 터치 & 드래그하면 여전히 카메라가 움직입니다.
    - WASD & Space 키를 누르면 여전히 캐릭터가 움직입니다.

**개발 환경을 알려주세요.
Window 11
M1 Mac Mini (MacOS 12.3)
iPhone SE (iOS 15.5)
iPad Pro (iPad OS 15.5)
Galaxy S8+ (Android 9)
Galaxy S9 (Android 10)

**구현하신 코드를 첨부해 주세요. 스크린샷이 있다면 첨부해 주세요.
1. 컨트롤러를 끄기 위한 조치를 했을 뿐 추가 코드를 작성하지 않았습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 카메라를 끄는 스크립트가 없으면 카메라가 움직이게 됩니다.
쿼터뷰 예제에 카메라가 고정되어 있고 드래그로 캐릭터가 움직이도록 되어 있으니 참고 부탁 드립니다.
키보드에 의한 이동은 유니티 에디터에서 테스트에 한합니다.
https://github.com/naverz/zepeto-multiplay-example/tree/main/Assets/Scenes/QuarterView
